api/IDBFactory.json

```diff
@@ -148,8 +148,7 @@
             "chrome_android": "mirror",
             "edge": "mirror",
             "firefox": {
-              "version_added": false,
-              "impl_url": "https://bugzil.la/934640"
+              "version_added": "126"
```

Comment:
This is good - related docs work in https://github.com/mdn/browser-compat-data/pull/22854